components/layout/Search.vue

```diff
@@ -1,38 +1,92 @@
+                                >
                                     <div class="result rounded-3">
                                         <div class="w-100">
                                             <h5>
-                                                <CommonTaskIcon v-if="result.cls" :cls="result.cls" />
-                                                <component class="icon-wrapper" v-else :is="iconByType(result.type)" />
+                                                <CommonTaskIcon
+                                                    v-if="result.cls"
+                                                    :cls="result.cls"
+                                                />
+                                                <component
+                                                    class="icon-wrapper"
+                                                    v-else
+                                                    :is="
+                                                        iconByType(result.type)
+                                                    "
+                                                />
                                                 {{ result.title }}
                                             </h5>
                                             <div class="slug">
                                                 <span
-                                                    :class="{first: index === 0}"
-                                                    v-for="(item, index) in breadcrumb(result.url)" :key="item">
-                                                        {{ item }}
+                                                    :class="{
+                                                        first: index === 0,
+                                                    }"
+                                                    v-for="(
+                                                        item, index
+                                                    ) in breadcrumb(result.url)"
+                                                    :key="item"
+                                                >
+                                                    {{ item }}
                                                 </span>
                                             </div>
-                                            <span class="badge bg-light">{{result.type.charAt(0).toUpperCase() + result.type.slice(1).toLowerCase()}}</span>
+                                            <span class="badge bg-light">{{
+                                                result.type
+                                                    .charAt(0)
+                                                    .toUpperCase() +
+                                                result.type
+                                                    .slice(1)
+                                                    .toLowerCase()
```

Comment:
I find the old inline style easier to read (simple string transform + concat like that can be one-lined imo)